### ISSUE TITLE
Coroutine Tracing Context Propagation Docs Outdated

### ISSUE THREAD

**Issue by scprek** [micronaut-projects/micronaut-core#9392]
### Issue description

I expect there to be information about how Tracing context is passed to coroutines and in the micronaut core docs there is a section https://docs.micronaut.io/latest/guide/index.html#coroutineTracingContextPropagation which references the class `CoroutineTracingDispatcher` that no longer exists. I assume this was due to the launch of https://micronaut-projects.github.io/micronaut-tracing/latest/guide/ and it's unclear to me if a context still needs to be added or not.

**Comment by doppleware** [comment]
Was this ever resolved?

**Comment by scprek** [comment]
I don’t think so, I ended up just writing extensions like this in a shared library for teams to use.

```kotlin

import io.opentelemetry.context.Context
import io.opentelemetry.extension.kotlin.asContextElement
import kotlin.coroutines.CoroutineContext
import kotlinx.coroutines.CoroutineScope
import kotlinx.coroutines.Dispatchers
import kotlinx.coroutines.runBlocking
import kotlinx.coroutines.slf4j.MDCContext


/**
 * Modifies the defaults of [runBlocking] by automatically including the [MDCContext] and [asContextElement] of OTEL
 * and using the Blocking Thread
 */
fun <T> runBlockingWithContext(context: CoroutineContext = Dispatchers.IO + MDCContext() + Context.current().asContextElement(), block: suspend CoroutineScope.() -> T): T {
    return runBlocking(context, block)
}

```

**Comment by doppleware** [comment]
Thanks for sharing this! I'm still trying to hack it into working, I'll report if I make a breakthrough ...

**Comment by sdelamo** [comment]
@dstepanov can you document this and fix the docs?

**Comment by graemerocher** [comment]
Triage: this remains a valid documentation task in Core. The current guide source still references the removed CoroutineTracingDispatcher/HttpCoroutineContextFactory API, while Micronaut 4 introduced the newer scoped context-propagation model and tracing integrations. Update the page to the supported Kotlin coroutine/tracing setup, link the micronaut-tracing guide where ownership moved, and include a current example showing how context is preserved when launching child coroutines from a regular function.
